onnxruntime/core/session/inference_session.cc

```diff
@@ -253,6 +254,36 @@ void InferenceSession::ConstructorCommon(const SessionOptions& session_options,
   // after the invocation of FinalizeSessionOptions.
   InitLogger(logging_manager_);  // this sets session_logger_ so that it can be used for logging after this point.
 
+  LOGS(*session_logger_, INFO) << "Session Options { "
```

Comment:
For your information, our logging implementation assumes all the byte strings are encoded in UTF-8. You may check ostream_sink.cc

Reply:
I am not sure if you are commenting on this aspect, but logging of paths is UTF-8 using that new macro I introduced. This is because on Windows "#define ORTCHAR_T wchar_t" but on other OSes it is char.

Reply:
It's about encoding.  All the strings you are dealing with here are multi-bytes strings, not Unicode strings.  Normally on Windows when you interact with Windows APIs with a multi-bytes string, the string needs be encoded in local ACP(like CP936 or something, but almost never UTF-8). However, in ONNX Runtime code we assume almost all std::string are UTF-8 strings.